gcp/make_bucket

```diff
@@ -24,4 +28,6 @@ fi
 
 info "Creating bucket in GCP. Disregard warnings about KMS permissions, our storage accounts are authorized."
 gsutil mb -c standard -b on -l europe-west4 -p ${project_name} gs://${bucket_name}
-gsutil kms encryption -w -k projects/hmf-database/locations/europe-west4/keyRings/hmf-database/cryptoKeys/hmf-database-20191001 gs://${bucket_name}
\ No newline at end of file
+gsutil kms encryption -w -k projects/hmf-database/locations/europe-west4/keyRings/hmf-database/cryptoKeys/hmf-database-20191001 gs://${bucket_name}
+gsutil label ch -l user:$USER gs://${bucket_name}
```

Comment:
Does `USER` work on Mac OS? Some shells don't set it. An alternative is the `whoami` command if needed.

Reply:
it does